app/models/workarea/payment/authorize/credit_card.decorator

```diff
@@ -44,6 +47,10 @@ module Workarea
       }
     end
 
+    def customer_ip_address
```

Comment:
What is this method used for?

Reply:
Ok, it looks like those methods are defined and used, like`customer_profile_id` is used here: https://github.com/workarea-commerce/workarea-authorize-cim/blob/master/app/models/workarea/payment/authorize/credit_card.decorator#L26

Let's either use them or take them out. Appreciate you trying to follow the patterns!

Reply:
Wow, I apologize Ben. Now I see why you mentioned it. I must have stopped using the variable at some point for debugging and forgot to put it back in place. This comment makes a lot more sense now :) It has been fixed!